docs/room_booking/features.md

```diff
@@ -1,3 +1,5 @@
+# Summary of Feature Set
+
 ### General
```

Comment:
Is it correct that we go from h1 directly to h3 titles?

Reply:
It's not wrong, I guess? Unless there's a rule for that. I can change all the h3 to h2

Reply:
usually you use h2 after h1, and h3 if you need another layer of nesting..